### DISCUSSION TITLE
Multiple Search Locations for Games

### DISCUSSION THREAD

**Discussion by rebornsunny2000** [skyline-emu/skyline#847] [Ideas]
I dunno if this has been discussed yet or if it has already been planned, but will we have an option to select multiple storage locations? Something like AetherSX2 where we can add a folder from the Internal Storage and another folder which may be in the SD Card?

**Comment by PixelyIon** [comment]
We feel the use cases for this would be too niche and it does involve some refactoring to properly implement which is why we don't plan to do so.
